SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Shared/NamespaceDoc.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.AppCenter.Analytics
     /// <summary>
     /// Analytics SDK module.
     /// </summary>
-    internal class NamespaceDoc
+    class NamespaceDoc
```

Comment:
why?

Reply:
There is a few such classes. It was only one with explicit one, so it's harmonization and aligning with documentation